cmd/mattermost/commands/config.go

```diff
@@ -68,15 +68,27 @@ var ConfigSetCmd = &cobra.Command{
 	RunE:    configSetCmdF,
 }
 
+var MigrateConfigCmd = &cobra.Command{
+	Use:   "migrate",
+	Short: "Migrate config file",
+	Long:  "Instantiates a new config store and sets it based on an existing defined store",
```

Comment:
Maybe its me, but the long description is a bit un-clear. @wiersgallak any opinion here?